packages/fxa-settings/README.md

```diff
@@ -1,6 +1,6 @@
 # Firefox Accounts Settings
 
-This documentation is up to date as of 2020-08-03.
+This documentation is up to date as of 2020-08-19.
```

Comment:
I see this quickly become a nuisance 😈

Reply:
Heh, honestly I like it for now to easily see the last time this doc was updated and checked for accuracy, but it'd be better if it was automated across all of our docs and it might not be ideal down the road when _parts_ of this doc are out of date...